### PR TITLE
[Cards][Addinformation][1431794]

### PR DESCRIPTION
**Pull request by Rajeshwari-v**
Added additional information in Note section-line no.74

### CODE REVIEW

**Review by Nikitha-MSFT — DISMISSED** [review]
Reviewed and Approved

File: msteams-platform/task-modules-and-cards/cards/cards-reference.md [hunk 1]
```diff
@@ -70,7 +70,8 @@ A customizable card that can contain any combination of text, speech, images, bu
 |
 
 > [!NOTE]
-> Media elements are currently not supported in Adaptive cards v1.2 on the Teams platform.
+> * Media elements are currently not supported in Adaptive cards v1.2 on the Teams platform.
+> * Teams platform supports up to version 1.2 of Adaptive card features.
```
Comment (KirtiPereira) [inline msteams-platform/task-modules-and-cards/cards/cards-reference.md:75]: MSTP does not support usage of 'up to'. Use earlier/later instead.
```Suggestion
> * Teams platform supports v1.2 or earlier of Adaptive card features.
> * Media elements are currently not supported in Adaptive card v1.2 on the Teams platform.
```
Please verify the above and update the note.

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
@Rajeshwari-v  - please verify and update the note accordingly.

**Review by Nikitha-MSFT — APPROVED** [review]
Reviewed and Approved.

**Review by KirtiPereira — APPROVED** [review]
Approved for merging.

**Review by surbhigupta12 — APPROVED** [review]
Approving the PR.

**Pull request merged**